### DISCUSSION TITLE
Scaling element height prior to calling html2canvas when height exceeds browser limits

### DISCUSSION THREAD

**Discussion by bchesley** [niklasvh/html2canvas#2679] [Q&A]
When trying to capture a large scrolling div (i.e., 68,000 px height) for saving as an image with jsPDF I am seeing inconsistent content output.  Sometimes clipped at the bottom or middle, sometimes no output at all in particular if I zoom the browser in or out (Chrome 92).

Any thoughts on scaling down the element itself prior to converting to canvas or other solutions are much appreciated.

**Comment by beipiaoyu2011** [comment]
same question. no solution. Maybe the solution is multiple screenshot merge

**Comment by royyin224** [comment]
I have solved a similar issue by specifying the scale and canvas's width and height so the output is a single image that contains all the content that I want from a certain HTML element

```
  function takeScreenShot(element) {
    html2canvas(element, {
      scale: 1,
      width: element.width,
      height: element.height
    }).then(function (canvas) {
      document.body.appendChild(canvas);
    });
  }
```
